style.css

```diff
@@ -0,0 +1,3 @@
+h1.heading {
+  color: blue;
+}
```

Comment:
- [ ] [Optional] I would suggest you please use the class or id to style the content, don't use `h1`,`h2`,`h3`,`p` tags because in more significant projects you will have many such tag and it will hamper the other content.